app/jobs/partnership_finalisation_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class PartnershipFinalisationJob < ApplicationJob
+  discard_on ActiveJob::DeserializationError
+
+  def perform(partnership_request)
+    return if partnership_request.reload.blank?
+
+    originator = partnership_request.versions.first.whodunnit
+    PaperTrail.request(whodunnit: originator) do
+      partnership_request.finalise!
+    end
```

Comment:
What does the `first` version mean? First ever created? Last ever created? First bu alphabetical order of our uids?

Reply:
papertrail orders its versions by created_at, something it calls "timestamp order"